src/search-injection/constants.js

```diff
@@ -18,7 +18,7 @@ export const SEARCH_ENGINES = {
     google: {
         regex: /(http[s]?:\/\/)?(www.)?google[.\w]+\/search\?.*/,
         container: {
-            above: 'center_col',
+            above: 'res',
```

Comment:
Appending to `res` makes Memex Container go below ads. Did you see that?

Reply:
changed it again, thanks for the headsup!